Neo4j.Driver/Neo4j.Driver.Tests.Integration/Extensions/CollectionExtensions.cs

```diff
@@ -29,5 +29,21 @@ public static T RandomElement<T>(this IEnumerable<T> list)
             var index = random.Next(0, list.Count());
             return list.ElementAt(index);
         }
+
+        public static IEnumerable<IEnumerable<T>> Batch<T>(this IEnumerable<T> collection, int batchSize)
+        {
+            var nextBatch = new List<T>(batchSize);
+            foreach (var item in collection)
+            {
+                nextBatch.Add(item);
+                if (nextBatch.Count != batchSize) continue;
```

Comment:
Maybe it would be easier to read without using `continue`?
```
if (nextBatch.Count == batchSize) 
{
    yield return nextBatch;
    nextBatch = new List<T>(batchSize);
}
```

Reply:
Yeah, that I followed Rider's suggestion to reduce nesting :)

I'll fix it in #353.